Applications/Games/Steam/Online/script.js

```diff
@@ -15,6 +15,8 @@ var installerImplementation = {
             .wineVersion(LATEST_STAGING_VERSION)
             .wineDistribution("staging")
             .preInstall(function (wine/*, wizard*/) {
+               wine.setOsForApplication().set("steam.exe", "winxp").do();
+               wine.setOsForApplication().set("steamwebhelper.exe", "winxp").do();
                wine.windowsVersion("winxp");
```

Comment:
This will set winxp for the whole prefix, if you only need it for two applications delete this

Reply:
ah mb, fixing it